eclipselink/src/test/groovy/jpql_sort/QuerySortTest.groovy

```diff
@@ -148,7 +148,10 @@ class QuerySortTest extends DataSpec {
 
         then:
 
-        thrown(UnsupportedOperationException)
+        queryBuilder.getResultQueryString() == 'select u from sec$User u order by u.login asc, u.name desc, u.id asc'
```

Comment:
The current test should change its name. The "sort by multiple properties in different directions is not supported" is not a valid one any more.

Reply:
fixed